base/reduce.jl

```diff
@@ -636,6 +636,31 @@ const ∈ = in
 ∋(itr, x)= ∈(x, itr)
 ∌(itr, x)=!∋(itr, x)
 
+"""
+    contains(fun, itr, x) -> Bool
+
+Determine whether there is any element `x` such that `fun(y,x)` is `true`, for all `y` in `itr`.
```

Comment:
This reads a bit funny. My first interpretation was `any(x -> all(y -> fun(x,y), itr), itr)`. Maybe "Determine whether there is any element `y` in `itr` such that `fun(y,x)` is `true`."?